aliases/available/terraform.aliases.bash

```diff
@@ -0,0 +1,11 @@
+# Aliases for Terraform and Terragrunt
+
+cite 'about-alias'
+about-alias 'Terraform abbreviations'
+
+alias tf='terraform'
```

Comment:
seems like there is a package named `tf`, but its not really used too much...
I guess it is okay

Reply:
@NoahGorny Yeah these always bug me, but assuming we ensure aliases stay conceptually grouped, the user should know what they're getting themselves into when they enable the alias group. That said, idea for the future: display aliases being set when toggling an alias (I'm sure this can be extended to plugins too).

Reply:
@NoahGorny do you think aliases should check for command existence?
```
_command_exists terraform || return 0
```

Reply:
> @NoahGorny do you think aliases should check for command existence?
> ```
> _command_exists terraform || return 0
> ```

I do not think it is necessary, it does not break anything new, so creating them is not really a problem in my opinion